docs/dsp-tools-xmlupload.md

```diff
@@ -854,8 +833,40 @@ Example:
 </region>
 ```
 
-Technical note: A `<region>` is in fact a `<resource restype="Region">`. But it is mandatory to use the 
+
+The circle and polygon are created with the following syntax:
+```json
+{
+    "status": "active",
+    "type": "circle",
+    "lineColor": "#ff1100",
+    "lineWidth": 5,
+    "points": [{"x":0.5,"y":0.3}],
+    "radius": {"x":0.1,"y":0.1}
```

Comment:
ah, I see now, this can be an ellipsis. But wouldn't it be simpler to define the ellipsis by the bounding rectangle? (like e.g. in paint, when you "drag" a box with the mouse, and it "fills" it with an ellipsis)

Reply:
No, I don't think  that an ellipsis is possible. It's just that the creators of knora decided to define the radius as vector (x, y). It's a bit like the hypothenuse of a right-angled triangle. I added a comment line in the JSON example to make it more clear.

Reply:
ok, I understand, thanks for the explanation. Interesting design choice. ;-)  
Then my point remains, that in my opinion it _should_ be an ellipsis, not a circle... (but again, that's not scope of this PR, so let's just keep it in the back of our heads)